src/main/kotlin/no/nav/syfo/melding/api/MeldingApi.kt

```diff
@@ -104,6 +105,38 @@ fun Route.registerMeldingApi(
 
             call.respond(HttpStatusCode.OK)
         }
+
+        post("/{$uuid}/retur") {
+            val personIdent = call.personIdent()
+            val veilederIdent = call.getNAVIdent()
+            call.checkVeilederTilgang(
+                action = API_ACTION,
+                veilederTilgangskontrollClient = veilederTilgangskontrollClient,
+                personIdent = personIdent
+            )
+
+            val meldingUuid = call.meldingUuid()
+            val innkommendeLegeerklaringMelding = meldingService.getMeldingFraBehandler(meldingUuid)
+                ?: throw IllegalArgumentException("Failed to create retur av legeerklæring: Melding with uuid $meldingUuid does not exist")
+            val opprinneligUtgaendeForesporsel = meldingService.getUtgaendeMeldingerInConversation(
+                conversationRef = innkommendeLegeerklaringMelding.conversationRef,
+                personIdent = innkommendeLegeerklaringMelding.arbeidstakerPersonIdent,
+            ).first()
```

Comment:
Vi trenger både `innkommendeLegeerklaringMelding` for å sette riktig `parentRef` på retur-meldingen, i tillegg til originalforespørselen for å sette en del andre felt (se `to.MeldingTilBehandler()`). 

I tillegg så er det `.first()` fordi det kan være påminnelser i denne lista med utgående meldinger også. Mulig man kunne vist dette mer eksplisitt ved feks `filter { it.type == MeldingType.FORESPORSEL_PASIENT_TILLEGGSOPPLYSNINGER }.first()`?